build/task-package.js

```diff
@@ -91,17 +72,16 @@ export default async function() {
   await builder.build({
     devMetadata: {
       directories: {
-        app: Const.ROOT,
+        app: Const.LIB_DIR,
         output: Const.PACKAGED_DIST_DIR,
       },
       build: {
         appId: 'tofino',
         'app-category-type': 'web',
-        asar: true,
-        ignore: IGNORE,
+        asar: false,
```

Comment:
Stock node can't handle asar packages so the services and all the node modules they depend on can't be in asar. The UI itself just webpacks down to a single file anyway so there isn't anything to gain from asar there.